### PR TITLE
docs: update MongoDB Atlas setup guide for local development (#6354)

### LINKED ISSUE #6354: [DOCS] <Adding a new docs page specific for using Atlas>
### Description

Hello,

I was in the process of setting up LinkFree on my local system to make some changes to the CSS. However, I encountered an issue when I realized that MongoDB is required to run the project locally, whereas I am more familiar with MySQL. I believe there may be others who are also unfamiliar with MongoDB but are interested in contributing to LinkFree.

To make it easier for potential contributors, I suggest adding a step-by-step guide in the project documentation that provides instructions for installing MongoDB locally with Atlas, as it offers a free tier. This would be helpful in getting new contributors up and running quickly and enabling them to contribute to the project with ease.

Thank you for considering my suggestion.

Best,
Veerpal

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by weilincheng**
## Fixes Issue
Closes #6354 

This PR addresses the issue of the missing MongoDB Atlas setup guide for LinkFree local development.

## Changes proposed
Adds a new page for the MongoDB Atlas setup guide for LinkFree local development.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6690"></a>

### CODE REVIEW

File: pages/docs/environments/local-with-atlas.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+1. Go to the MongoDB Atlas website (https://www.mongodb.com/cloud/atlas) and click on the "Try Free" button.
+2. Fill in your first/last name, email address, and password, and click "Create your Atlas account".
+3. Check your inbox and click the verification link inside to complete your registration.
+
+### Deploy a New Database
+
+1. Once you've completed the registration, you will be redirected to a page to deploy your database.
+2. Choose M0 for the cluster tier. For local development, the M0 free tier is sufficient.
+3. Choose the provider and region based on your preference. Ideally, choose the region closest to your location.
+   ![image](https://user-images.githubusercontent.com/7236196/236708986-eb643efc-78cb-46b2-9aa9-28a83b8133ef.png)
```
Comment (eddiejaoude) [inline pages/docs/environments/local-with-atlas.mdx:21]: Just tweaking the formatting

```suggestion
3. Choose the provider and region based on your preference. Ideally, choose the region closest to your location.

![image](https://user-images.githubusercontent.com/7236196/236708986-eb643efc-78cb-46b2-9aa9-28a83b8133ef.png)
```

File: pages/docs/environments/local-with-atlas.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+1. Once you've created the database, you will be redirected to the security quickstart pages.
+2. Fill in the username and password, and then click the "Create User" button to create your first database user.
+   Note: Save your password securely, as MongoDB Atlas doesn't store it.
+3. Click the "Add My Current IP Address" button to add an entry to your IP access list. Only an IP address you add to the IP access list will be able to connect to your database.
+4. Click the "Finish and Close" button.
+
+### Set Up the Database Connection
+
+1. Once your cluster is ready, click the "Connect" button on your Database Deployments console.
+   ![image](https://user-images.githubusercontent.com/7236196/236709649-4fe1dd9e-fbd1-4df3-b0ff-a845069496b3.png)
```
Comment (eddiejaoude) [inline pages/docs/environments/local-with-atlas.mdx:35]:
```suggestion
![image](https://user-images.githubusercontent.com/7236196/236709649-4fe1dd9e-fbd1-4df3-b0ff-a845069496b3.png)
```

File: pages/docs/environments/local-with-atlas.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+2. Click "Drivers" under the Connect to your application section.  
+   ![image](https://user-images.githubusercontent.com/7236196/236709974-5e8a1c36-2edb-4a56-bc0d-7ab3c9bd4019.png)
```
Comment (eddiejaoude) [inline pages/docs/environments/local-with-atlas.mdx:38]:
```suggestion

![image](https://user-images.githubusercontent.com/7236196/236709974-5e8a1c36-2edb-4a56-bc0d-7ab3c9bd4019.png)
```

**Review by eddiejaoude — COMMENTED** [review]
Looks great 👍 , just some minor formatting tweaks

**Comment by weilincheng** [comment]
> Looks great 👍 , just some minor formatting tweaks

Eddie, thank you for the detailed review. 💯 
I have made the modifications that you suggested and pushed the changes to the repository. Please let me know if there is anything else that needs to be addressed. 

This is my first time creating a draft pull request. When should I change it to an official pull request?

**Comment by eddiejaoude** [comment]
Looks great 👍 Thank you!

About pull request status `draft`, it is for when you are still working on it, I think we could have made it `ready for review` sooner, but it is ok, I will change the status now and we can get it merge 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you for the contribution 👍

**Comment by weilincheng** [comment]
I'm happy to see my contribution is now online on LinkFree! 🚀
Thank you for your guidance during this process - I learned a lot from contributing to this project. 👍

**Comment by eddiejaoude** [comment]
Awesome to hear that! We really appreciate you adding value to the LinkFree project 🚀

**Pull request merged**